HW4_Shtompel/protein_tools.py

```diff
@@ -0,0 +1,200 @@
+"""
+Global variables:
+- AA_ALPHABET — a dictionary variable that contains a list of proteinogenic aminoacids classes.
+- ALL_AMINOACIDS — a set variable that contains a list of all proteinogenic aminoacids.
+- AMINO_ACIDS_MASSES — a dictionary variable that contains masses of all proteinogenic aminoacids.
+"""
+
+AA_ALPHABET = {'Nonpolar': ['G', 'A', 'V', 'I', 'L', 'P'],

```

Comment:
Здорово что правильно это дело оформили!

Reply:
Хотя это ж не алфавит)
Это группы аминокислот.
```suggestion
AA_GROUPS = {'Nonpolar': ['G', 'A', 'V', 'I', 'L', 'P'],

```